src/rnpkeys/rnpkeys.cpp

```diff
@@ -68,6 +68,12 @@ const char *usage =
   "  --revoke-key           Revoke a key specified.\n"
   "  --remove-key           Remove a key specified.\n"
   "  --edit-key             Edit key properties.\n"
+  "    --add-subkey         Add new subkey.\n"
+  "        --expert         Select subkey type, size and additional parameters.\n"
```

Comment:
Sub-options would duplicate the --generate-key options, making this list too long. Probably we should remove them, allowing user to read man in case he needs it.